packages/realm-server/billing/stripe-webhook-handlers/checkout-session-completed.ts

```diff
@@ -0,0 +1,43 @@
+import { DBAdapter } from '@cardstack/runtime-common';
+import {
+  insertStripeEvent,
+  markStripeEventAsProcessed,
+  updateUserStripeCustomerId,
+} from '../billing-queries';
+import { StripeCheckoutSessionCompletedWebhookEvent } from '.';
+
+import PgAdapter from '../../pg-adapter';
+import { TransactionManager } from '../../pg-transaction-manager';
+
+export async function handleCheckoutSessionCompleted(
+  dbAdapter: DBAdapter,
+  event: StripeCheckoutSessionCompletedWebhookEvent,
+): Promise<Response> {
+  let txManager = new TransactionManager(dbAdapter as PgAdapter);
+
+  await txManager.withTransaction(async () => {
+    try {
+      await insertStripeEvent(dbAdapter, event);
+    } catch (error) {
```

Comment:
Let's move this catch block into `insertStripeEvent` so we don't repeat the same code in all stripe handlers

Reply:
Saw it has been changed on recent PR, so I remove catch block instead